ui/src/app/pages/profile/profile-component.tsx

```diff
@@ -226,9 +227,12 @@ export const ProfileComponent = fp.flow(
       )(profile.accessModules.modules);
       const hasExpired = profileExpiration && profileExpiration < Date.now();
 
-      const urlError = professionalUrl
-      ? validate({website: professionalUrl}, {website: {url: {message: '^Professional URL %{value} is not a valid URL'}}})
-      : undefined;
+      const urlError =
+        professionalUrl ?
+        validate(
+          {website: canonicalizeUrl(professionalUrl)},
+          {website: {url: {message: '^Professional URL %{value} is not a valid URL'}}})
```

Comment:
opt nit: can produce some oddly formatted text, but still gets the message across

Example: `httpss://broad.net` renders here as `http://httpss://broad net` (note that a space is substituted for the dot)

Reply:
same comment for account creation

Reply:
Thanks, good catch - fixed.